### PR TITLE
add support for config option in http-servlet.

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

**Comment by rghetia** [comment]
refer to #1749 for the usage of config option.

**Comment by songy23** [comment]
Please add this change to CHANGELOG.

File: contrib/http_servlet/src/test/java/io/opencensus/contrib/http/servlet/OcHttpServletFilterTest.java [hunk 1]
```diff
@@ -53,20 +70,91 @@
...
+
+    List<String> attributes = stringArgumentCaptor.getAllValues();
+    assertThat(attributes.contains(OC_TRACE_PROPAGATOR)).isTrue();
+    assertThat(attributes.contains(OC_EXTRACTOR)).isTrue();
+    assertThat(attributes.contains(OC_PUBLIC_ENDPOINT)).isTrue();
+    assertThat(filter.handler).isNotEqualTo(oldHandler);
     assertThat(filter.handler).isNotNull();
   }
 
+  void testInitInvalidAttr(String attr) throws ServletException {
```
Comment (dinooliva) [inline contrib/http_servlet/src/test/java/io/opencensus/contrib/http/servlet/OcHttpServletFilterTest.java]: nit: consider making this private.

Comment (rghetia) [reply contrib/http_servlet/src/test/java/io/opencensus/contrib/http/servlet/OcHttpServletFilterTest.java]: done.

**Comment by rghetia** [comment]
fixes #1674

**Pull request merged**